.github/workflows/cd.yaml

```diff
@@ -0,0 +1,71 @@
+name: CD
```

Comment:
waaaaait - not what I meant! Just remove the publish step from `ci.yaml` and we are good to do!

Reply:
AHHH Okay